framework/src/modules/chain/modules/multisignatures.js

```diff
@@ -176,10 +176,11 @@ __private.validateSignature = (
 	transaction.ready = library.logic.multisignature.ready(transaction, sender);
 
 	// Emit events
-	library.network.io.sockets.emit(
-		'multisignatures/signature/change',
+	library.channel.publish(
+		'chain:multisignatures:signature:change',
```

Comment:
@MaciejBaj Are we agreed on this convention to namespace events with `:` or we stick to old convention of `/` and use `:` only for module name separator?